src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.nuspec

```
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright (c) Microsoft. All rights reserved.
+Licensed under the MIT license. See LICENSE file in the project root for full license information.
+-->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>WebJobs.Extensions.DurableTask</id>
+    <!-- Needed to specify the version here for the nuspec to be valid -->
+    <version>$version$</version>
+    <title>Azure Functions Durable Task Extension</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <projectUrl>https://github.com/Azure/azure-functions-durable-extension</projectUrl>
+    <description>Azure WebJobs SDK Extension for the Durable Task Framework</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>azure,functions,azure-function,durable-functions</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.6.1" />
+      <group targetFramework=".NETStandard2.0" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src=".\bin\$configuration$\net461\publish\**" target="lib/net461"/>
+    <file src=".\bin\$configuration$\netstandard2.0\publish\**" target="lib/netstandard2.0"/>
+    <file src="..\..\_manifest\**" target="content/manifest"/>
```

Consider a more descriptive folder name e.g.
```suggestion
    <file src="..\..\_manifest\**" target="content/SBOM"/>
```